src/os/alert/alert.js

```diff
@@ -1,41 +1,48 @@
-goog.provide('os.alert.Alert');
-goog.require('os.alert.AlertEventSeverity');
+goog.module('os.alert.Alert');
+goog.module.declareLegacyNamespace();
 
+const AlertEventSeverity = goog.require('os.alert.AlertEventSeverity'); // eslint-disable-line no-unused-vars
```

Comment:
Is this require needed if this file only uses it as a type declaration, but never actually references? Seems annoying to have to require it like this and work around the linter.

Reply:
Yes. The Closure compiler will not understand the type unless required in the file. It's possible it would be implicitly required like a lot of our code is doing now, but you would have to use `os.alert.AlertEventSeverity` instead and that will likely stop working once we switch to ES6.

Related: https://github.com/google/closure-compiler/issues/3041